event/event.go

```diff
@@ -15,6 +15,25 @@ import (
 	"github.com/libp2p/go-libp2p/core/peer"
 )
 
+type Bus interface {
```

Comment:
todo: This is a user-public type, and should really have type level documentation, as well as the stuff on the functions.

Same for `Subscription`